jobs/telemetry_batch_view.py

```diff
@@ -1,33 +1,30 @@
 #!/usr/bin/env python
 
+from requests
 from os import chdir
 from os import environ
 from subprocess import call
 
 
+artifact_file = "artifact.jar"
+
+
 def call_exit_errors(command):
     print("+ {}".format(" ".join(command)))
     rc = call(command)
     if rc > 0:
         exit(rc)
 
 
-def clone_and_compile_tbv():
-    command = [
-        "git",
-        "clone",
-        environ.get("GIT_PATH", "https://github.com/mozilla/telemetry-batch-view.git"),
-        "--branch",
-        environ.get("GIT_BRANCH", "master"),
-    ]
-
-    call_exit_errors(command)
+def retrieve_jar():
+    jar_url = environ.get("ARTIFACT_URL")
```

Comment:
I think we should `retreive_jar` only when `GIT_PATH` and `GIT_BRANCH` are defaults, otherwise we should clone and compile.

Reply:
We already handle that case with the deploy script. Engineers can just push to a branch on `mozilla/$SLUG` and that branch will get built. Locally they can test Airflow using that artifact.

Instead, let's add an optional `BRANCH` or `TAG` (not both) which will let engineers specify builds for their jobs. `BRANCH` will point to an auto-updated artifact, but then to update jobs that use a previous release, the engineer will have to update their Airflow job manually (kind of annoying).

Reply:
I want to move away from the build and compile completely, and focus on just using built jars.